code/datums/trading/unique.dm

```diff
@@ -25,6 +25,64 @@
 /datum/trader/ship/unique/what_do_you_want()
 	return get_response("what_want", "I don't want anything!")
 
+/datum/trader/ship/unique/syndicate
```

Comment:
Unsure if that is a good idea

Reply:
We're already removing one unique store, and if a merchant wants to buy weapons they have much easier time buying them from conventional gunstores. Syndicate store only accepts payments in valuable goods from across the station, and as such would be very difficult to trade for by a non-traitor. Furthermore, the most dangerous items - Mauler + rigs - are all agent id locked. 

That said, I've had a discussion about this - the ai core and ai law boards might be too easy to make for traitors, essentially allowing to cheat their way into the system by sneaking into science with two backpacks and then buying everything they want. Do you think removing the ai boards would be a good suggestion? Maybe replace them with some super expensive items from top-end of the research tree? [example: Experimental KA core and Energy Glaive]

Reply:
Resolved with the revert of the multiplier.
We´ll see how the syndie borg is going to work out.